mne/channels/tests/test_montage.py

```diff
@@ -163,16 +163,13 @@ def test_read_dig_montage():
                                transform=True, dev_head_t=True)
     # check coordinate transformation
     # nasion
-    assert_almost_equal(montage.elp[0, 0], 0)
     assert_almost_equal(montage.nasion[0], 0)
-    assert_almost_equal(montage.elp[0, 2], 0)
     assert_almost_equal(montage.nasion[0], 0)
     # lpa and rpa
-    assert_allclose(montage.elp[1:3, 1:], 0, atol=1e-16)
     assert_allclose(montage.lpa[1:], 0, atol=1e-16)
     assert_allclose(montage.rpa[1:], 0, atol=1e-16)
     # device head transform
-    dev_head_t = fit_matched_points(tgt_pts=montage.elp[3:],
```

Comment:
these were checking for the fiducials. as @Eric89GXL pointed out, the intention with `elp` was to have the points in native digitizer space that match the points in the device space. these tests redundant the ones below it.